lib/improver/tests/spotdata/spotdata/test_ancillaries.py

```diff
@@ -39,9 +39,12 @@
 import iris
 from iris.coords import DimCoord
 from iris.cube import Cube
+from iris import FUTURE
 
 from improver.spotdata.ancillaries import get_ancillary_data as Plugin
 
+FUTURE.netcdf_no_unlimited = True
```

Comment:
Again should we have this in a with statement?

Reply:
Changed